core/shared/src/main/scala/zio/internal/FiberContext.scala

```diff
@@ -683,10 +685,20 @@ private[zio] final class FiberContext[E, A](
     if (exitAsync(epoch)) evaluateLater(zio)
   }
 
-  def interruptAs(fiberId: Fiber.Id): UIO[Exit[E, A]] = kill0(fiberId)
+  private def withChildren[A](f: java.util.Set[FiberContext[Any, Any]] => A): A =
```

Comment:
Is there any perf improvement that could be made by making this function return `Any`?

Reply:
I think it would roughly be the same...